shared/templates/static/ansible/no_empty_passwords.yml

```diff
@@ -3,9 +3,11 @@
 # strategy = configure
 # complexity = low
 # disruption = medium
-
 - name: "Prevent Log In to Accounts With Empty Password"
   replace:
     dest: /etc/pam.d/system-auth
     regexp: 'nullok\s*'
     replace: ''
+  tags:
```

Comment:
Almost all the tasks need ``check_mode: no`` ... didn't want to litter the PR with multiple of the same comment.